code/__DEFINES/traits.dm

```diff
@@ -882,3 +882,6 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 
 /// Ignores body_parts_covered during the add_fingerprint() proc. Works both on the person and the item in the glove slot.
 #define TRAIT_FINGERPRINT_PASSTHROUGH "fingerprint_passthrough"
+
+/// The machine has been upgraded with metallic hydrogen to halves the power consumption.
+#define TRAIT_MACHINE_POWER_UPGRADED "machine_power_upgraded"
```

Comment:
```suggestion
#define TRAIT_METAL_HYDROGEN_UPGRADED "metal_hydrogen_upgraded"
```
`machine_power_upgraded` is very generic, and this trait is only applied in a specific situation